hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -111,19 +109,8 @@ public void doStateTransition() {
             AccountID sponsor = txnCtx.activePayer();
 
             CryptoCreateTransactionBody op = cryptoCreateTxn.getCryptoCreateAccount();
-            long balance = op.getInitialBalance();
             final var customizer = asCustomizer(op);
-            final var isLazyCreation =
-                    !op.getAlias().isEmpty() && op.getAlias().size() == EVM_ADDRESS_SIZE && !op.hasKey();
-            final var lazyCreationFinalizationFee = autoCreationLogic.getLazyCreationFinalizationFee();
-            final var minPayerBalanceRequired = balance + (isLazyCreation ? lazyCreationFinalizationFee : 0);
-            if (minPayerBalanceRequired > (long) ledger.getAccountsLedger().get(sponsor, BALANCE)) {
-                throw new InsufficientFundsException(txnCtx.activePayer(), minPayerBalanceRequired);
-            }
-            final var created = ledger.create(sponsor, balance, customizer);
-            if (isLazyCreation) {
-                transferLogic.payAutoCreationFee(lazyCreationFinalizationFee);
-            }
+            final var created = ledger.create(sponsor, op.getInitialBalance(), customizer);
```

Comment:
With the hollow account logic deleted from here, the `autoCreationLogic` and `transferLogic` become unnecessary, I reckon we can delete them.

We can also revert the visibility of `TransferLogic#payAutoCreationFee` to `private`. The same goes for `AbstractAutoCreationLogic#getLazyCreationFinalizationFee`.

Reply:
Hm, interesting that Sonar did not complain about the unused `autoCreationLogic` and `transferLogic`, remove them as suggested.